src/definitions/modules/nag.js

```diff
@@ -181,39 +246,30 @@ $.fn.nag = function(parameters) {
             var
               options = module.get.storageOptions()
             ;
-            if(settings.storageMethod == 'localstorage' && window.localStorage !== undefined) {
-              window.localStorage.setItem(key, value);
-              module.debug('Value stored using local storage', key, value);
-            }
-            else if(settings.storageMethod == 'sessionstorage' && window.sessionStorage !== undefined) {
-              window.sessionStorage.setItem(key, value);
-              module.debug('Value stored using session storage', key, value);
+            if(storage === window.localStorage && options.expires) {
+              module.debug('Storing expiration value in localStorage', key, options.expires);
+              storage.setItem(key + settings.expirationKey, options.expires );
```

Comment:
[Cookie also supports expiration](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#Example_3_Do_something_only_once) in the form of `; expires=date-in-GMTString-format`. 
How about supporting expiration on cookie storage ?

Reply:
Cookie expiration is already built in via the given cookie options.

I only added the extra expiration here for localstorage, because localstorage does not expire values by default